Backpack/Carousel/Classes/BPKCarousel.swift

```diff
@@ -26,11 +26,7 @@ public protocol BPKCarouselDelegate: AnyObject {
 public final class BPKCarousel: UIView {
     private let internalCarousel: BPKInternalCarousel
     
-    public weak var delegate: BPKCarouselDelegate? {
-        didSet {
-            internalCarousel.delegate = self
-        }
-    }
```

Comment:
I think you'd still want to do this if someone sets a delegate for this.

Reply:
My understanding is the `internalCarousel.delegate` would never change after the Carousel is initialized.

When the external delegate (BPKCarouselDelegate) is updated, the function below calls it 

```
 public func onImageChange(index: Int) {
        pageIndicator.currentIndex = index
        delegate?.onImageChange(index: index)
    }
```

Reply:
Yeah. I reverted my change The didSet block is unchanged now